security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+Nodes are simply not incentivized to close triangles as they could have just routed payments instead of opening a new payment channel.
+Surprisingly, triangle closing is a common practice in the LN.
+The number of triangles was steadily growing before the implementation of multi-part payments.
+This is counterintuitive and surprising given that nodes could have just routed payments through the two sides of the triangle instead of opening the third channel.
+This may mean that routing inefficiencies incentivized users to close triangles and not fall back on routing.
+Hopefully, multi-part payments will help increase the effectiveness of payment routing.
+
+==== Centralization in the LN
+A common metric to assess the centrality of a node stem:[v] in a graph is its _betweenness centrality_.
+Let stem:[\sigma_{st}] denote the number of shortest paths between node stem:[s] and stem:[t].
+Similarly, stem:[\sigma_{st}(v)] denotes the number of the shortest paths between stem:[s] and stem:[t], that also pass through stem:[v].
+Intuitively, the more shortest paths pass through a certain node stem:[v], the more important and central stem:[v] is.
+Therefore, the ratio stem:[\frac{\sigma_{st}(v)}{\sigma_{st}}] is indicative of stem:[v]'s centrality.
+Betweenness centrality is defined to be the sum of all these ratios for every pair of nodes stem:[s] and stem:[t].
+More formally, the betweenness centrality a node stem:[g(v)] of a node stem:[v] is defined as stem:[g(v)=\sum_{s\ne v \ne t }\frac{\sigma_{st}(v)}{\sigma_{st}}].
+Central point dominance is a metric derived from betweenness centrality, used to assess the centrality of a network.
+For a precise definition of central point dominance, the reader is referred to https://doi.org/10.2307/3033543[Freeman's work].
+The larger the central point dominance of a network is, the more centralized the network is.
+We can observe that the LN has a greater central point dominance, ie. it is more centralized, than a random graph (Erdős-Rényi graph) or a scale-free graph (Barabási-Albert graph) of equal size.
+However, we also note, that over time the central point dominance of the LN gradually decreases as the network grows and densifies.
+
+[[central-point-dominance-ln]]
+.Central point dominance of the LN, a random graph (ER) and a scale-free graph (BA) of equal size.
+image:images/central-point-dominance.png[]
+
+In general, our understanding of the dynamic nature of the LN channel graph is rather limited.
+It is fruitful to analyze how protocol changes like multi-part payments can affect the dynamics of the LN.
+It would be beneficial to explore the temporal nature of the LN graph in more depth.
+
+
+=== Economic incentives and graph structure
+The Lightning graph forms spontaneously.
+Nodes connect to each other based on mutual interest.
+As a result, incentives drive graph development.
+Let's describe some of the relevant incentives:
+
+  * Rational incentives.
```

Comment:
A new incentive comes in the form of Lightning Pool: aside from the _potential_ to earn routing fees, nodes are also able to earn passive yield simply by allocating capital to the network where it's most demanded.